components/Modal/Modal.module.scss

```diff
@@ -0,0 +1,277 @@
+.overlay {
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: rgba(0, 0, 0, 0.8);
+  background-size: contain;
+  background-repeat: no-repeat;
+  z-index: 100;
+  position: fixed;
+  top: 0;
+  left: 0;
+  overflow: auto;
+  font-family: 'DM Sans', sans-serif;
+}
+
+.close {
+  position: absolute;
+  top: 0;
+  right: 0;
+  height: fit-content;
+  width: fit-content;
+  background: none;
+  border: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 18px;
+  color: #ffffff;
+  font-weight: 700;
+  font-size: 1.5rem;
+  font-family: 'DM Sans', sans-serif;
+  cursor: pointer;
+  padding: 0.25rem 0 0.25rem 0.25rem;
+  margin: 1.25rem 1.25rem 0 0;
+  transition: 0.2s opacity;
+
+  &:hover {
+    opacity: 0.5;
+  }
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  background-color: #ffffff;
+  box-shadow: 0 0 20px rgba(0, 0, 0, 0.2);
+  height: fit-content;
+  min-width: 288px;
+  border-radius: 20px;
+  width: calc(100% - 2.5rem);
+  padding: 1.25rem;
+  margin: 4.5rem 1.25rem 1.25rem 1.25rem;
+  overflow-y: auto;
+
+  h2 {
+    color: #000000;
+    font-size: 1.5rem;
+    font-weight: 700;
+    margin-bottom: 0.5rem;
+  }
+
+  a {
+    font-weight: 400;
+    color: #3107eb;
+  }
+
+  p {
+    margin: 0;
+  }
+
+  > p {
+    margin-bottom: 2rem;
+  }
+
+  p > span {
+    font-weight: 700;
+  }
+}
+
+.headings {
+  margin-bottom: 1rem;
+
+  h1 {
+    font-size: 3rem;
+    margin-bottom: 0.5rem;
+  }
+
+  p {
+    font-size: 18px;
+    margin-bottom: 0.5rem;
+  }
+
+  span {
+    display: inline-block;
+    padding: 0.5rem 0.75rem;
+    border-radius: 0.25rem;
+    font-weight: 700;
+    font-size: 14px;
+    color: #ffffff;
+  }
+}
+
+.imageContainer {
+  position: relative;
+  margin: auto;
+  margin-bottom: 1rem;
+  max-width: 475px;
+
+  .product {
+    object-fit: contain;
+    width: 100%;
+    border-radius: 10px;
+  }
+
+  .badge {
+    position: absolute;
+    top: 0;
+    right: 0;
+    width: 79.9px;
+    height: 71.5px;
+    object-fit: contain;
+  }
+}
+
+.sourcing {
+  margin-bottom: 1.5rem;
+
+  > p:first-of-type {
+    margin-bottom: 1rem;
+  }
+
+  .linkLeadInfo {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    white-space: nowrap;
+    margin-bottom: 1rem;
+
+    a {
+      display: inline-block;
+      min-width: 162px;
+      width: fit-content;
+      max-width: 100%;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      margin-right: 0.5rem;
+    }
+
+    p {
+      display: inline-block;
+      margin: 0;
+    }
+
+    span {
```

Comment:
heyo - in general, please avoid tag/type/dangling element selectors (https://medium.com/@reemtos/dangling-element-selectors-in-css-and-why-you-should-avoid-them-171f6cd989b1) 

certain ones are more fine if they're a little bit semantic - i.e. `em` or `strong` as they describe emphasis or the strong tag, but `span` is just very generic (and if I did that in the past whoops you can blame me haha). but if possible try to come up with semantic classnames

Reply:
Awesome! Thanks for the resource. I left a couple in there as resets for anchor and p tags, otherwise I got it pretty cleaned up.